core/shared/src/main/scala/zio/prelude/AssociativeEither.scala

```diff
@@ -214,28 +212,24 @@ object AssociativeEither extends LawfulF.Invariant[AssociativeEitherDeriveEqualI
     }
 
   /**
-   * The `AssociativeEither` instance for `ZStream`.
+   * The `CommutativeEither` (and `AssociativeEither`) instance for `ZStream`.
    */
-  implicit def ZStreamAssociativeEither[R, E]: AssociativeEither[({ type lambda[+a] = ZStream[R, E, a] })#lambda] =
-    new AssociativeEither[({ type lambda[+a] = ZStream[R, E, a] })#lambda] {
+  implicit def ZStreamCommutativeEither[R, E]: CommutativeEither[({ type lambda[+a] = ZStream[R, E, a] })#lambda] =
+    new CommutativeEither[({ type lambda[+a] = ZStream[R, E, a] })#lambda] {
       def either[A, B](fa: => ZStream[R, E, A], fb: => ZStream[R, E, B]): ZStream[R, E, Either[A, B]] =
-        fa.map(Left(_)) orElse fb.map(Right(_))
+        fa mergeEither fb
     }
 
   /**
-   * The `AssociativeEither` instance for failed `ZStream`.
+   * The `CommutativeEither` instance for `ZSink`.
```

Comment:
> (and `AssociativeEither`)

?
:slightly_smiling_face:

Reply:
Oh you're right! , let me do the change.